src/client/public/runner.ts

```diff
@@ -392,13 +384,12 @@ interface MakerInitializationParams {
                     $('.runner-overlay').hide();
                     $anotherSnippetSelected.hide();
                 } else {
-                    if (isListeningTo.snippetSwitching) {
-                        $anotherSnippetSelected.find('.ms-MessageBar-text .snippet-name').text(input.message.name);
-                        showReloadNotification($anotherSnippetSelected,
-                            () => clearAndRefresh(input.message.id, input.message.name, false /*isTrustedSnippet*/),
-                            () => isListeningTo.snippetSwitching = false,
-                            true /*allowShowLoadingDots*/);
-                    }
+                    // TODO: remove snippetSwitching if possible
```

Comment:
Let's test this on your machine.

Reply:
At minimum we should get rid of it entirely if we decide we truly don't need it.